functions/exercise2/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>excercise2</title>
+    <script src="script.js"></script>
+</head>
+
+<body>
+    <h1>Palindrome</h1>
+    Phrase:<input id="phrase" />
+    <button onclick="showResult()">Click here to check if your phrase is palindrome</button> Result:
+    <input type="text" disabled="disabled" size="60" id="result" />
```

Comment:
You did not refer to my comment: "Why this field is input field? It prompts someone to put a value here."